gradle-junit-reports/src/test/java/com/palantir/gradle/junit/TestCommon.java

```diff
@@ -1,11 +1,11 @@
 /*
- * (c) Copyright 2017 Palantir Technologies Inc. All rights reserved.
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
```

Comment:
All of these _should_ be preserved when https://github.com/palantir/gradle-baseline/pull/1227 lands